pr_agent/algo/file_filter.py

```diff
@@ -0,0 +1,22 @@
+import fnmatch
+import re
+
+from pr_agent.config_loader import get_settings
+
+def filter_ignored(files):
```

Comment:
please wrap this in: 
"try-except"

Surprises can happen, and its always good to add some protection for a new feature

Reply:
Done, fixed in 92e9012fb63d21f2464f48979c9e642a43e9d6ed. Test case also provided.